### PR TITLE
Add purchase product barcode

### PR DESCRIPTION
**Pull request by TimLai125**

### CODE REVIEW

File: purchase_product_barcode/models/purchase_order_line.py
Comment (TimLai125) [inline purchase_product_barcode/models/purchase_order_line.py]: Considering about readonly attribute to this field.

File: purchase_product_barcode/reports/purchase_order_templates.xml
Comment (TimLai125) [inline purchase_product_barcode/reports/purchase_order_templates.xml:14]: Odoo does not support 'JAN' barcode directly but 'JAN' is compatible with 'EAN13'.

**Comment by TimLai125** [comment]
The PR is good to review.

File: purchase_product_barcode/models/purchase_order_line.py
Comment (yostashiro) [reply purchase_product_barcode/models/purchase_order_line.py]: Yes, please make the field readonly.

**Review by yostashiro — CHANGES_REQUESTED** [review]
@TimLai125 please merge this PR once you have made an update on barcode field.  Thanks!

**Comment by TimLai125** [comment]
@yostashiro I do not have the write access to this repository to merge the PR

**Comment by yostashiro** [comment]
> I do not have the write access to this repository to merge the PR

@TimLai125 Try again.  I have updated the team settings.

**Pull request merged**
